index.html

```diff
@@ -71,5 +73,217 @@ <h3 class="profession">A Web Developer</h3>
         </div>
       </div>
     </section>
+    <section class="work-section">
+      <header class="work-section-head">
+        <div class="my-recent-works">My Recent Works</div>
+      </header>
```

Comment:
Kindly, by convention your website should only have 1 HEADER section, which is the top bar where the NAVBAR is located, I recommend changing it to a DIV tag

Reply:
ok